README.md

```diff
@@ -22,14 +25,73 @@ syncClient.init({
     storage_strategy: ['memory'],
     crashed_count_wait: 0
 });
+
 ```
 
+
 ## Building
 
     npm install
     grunt 
 
-## Relation for fh-js-sdk
 
-Feedhenry JS SDK contains various libraries and also includes fh-sync-js library.
+## Example App
+
+The ../dist/fh-sync.js is create with grunt and is the runtime application that is called duing client sync. To run the client open _./example/index.html_ in a browser when the server
+ is running. You won't see anything running in the browser until you check the developer tools (ctrl/shift/i)
+Check the network tabs and console tabs to see sync in action. 
+For more information on browser developer tools see
+
+https://developer.chrome.com/devtools
+
+https://developer.mozilla.org/en-US/docs/Tools/Tools_Toolbox
+
+
+## Some Available Typescript sync API methods
```

Comment:
Duplication of the methods automatically generated as api (see ticket for api generator) and typescript definitions. Definetly not for a readme file.